### DISCUSSION TITLE
6C1BE571-3CB8-4E9B-9215-DA7BC6FFF768

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#104] [General]
# 6C1BE571-3CB8-4E9B-9215-DA7BC6FFF768

Week 35, 2023: India Becomes Fourth Nation to Land on the Moon

https://celestia.mobi/article?guide=6C1BE571-3CB8-4E9B-9215-DA7BC6FFF768&lang=zh_CN

**Comment by 13940951572** [comment]
这个航天器这个插件有吗？

**Comment by BashaDey** [comment]
Bravo India!!!

**Reply by javietxebarria** [reply]
Good job India!

**Comment by javietxebarria** [comment]
Good job India!
